website/addons/twofactor/models.py

```diff
@@ -52,8 +52,8 @@ def verify_code(self, code):
 
     def on_add(self):
         # TODO(hrybacki, sloria): push status message shouldn't need a session
-        push_status_message('Please <a href="#TfaVerify">activate your'
-                            ' device</a> before continuing.')
+        push_status_message('<span id="tfa-activation-warning">Please <a href="#TfaVerify">activate your'
```

Comment:
IDs should be in the following format: `TfaActivationWarning`

ref: http://cosdev.readthedocs.org/en/latest/style_guides/html_css.html#html-and-css